__tests__/app.test.js

```diff
@@ -16,51 +16,98 @@ beforeEach(() => {
+          expect(body.length).toBe(3);
         });
-        test("Should return an array with length equal to total number of topics in database", () => {
-            return request(app)
-                .get("/api/topics")
-                .expect(200)
-                .then(({ body }) => {
-                    expect(body.length).toBe(3);
-                });
+    });
+    test(`Each object in the  returned array should have keys of 'slug' and 'description'`, () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((topic) => {
+            expect(topic).toHaveProperty("slug");
+            expect(topic).toHaveProperty("description");
+          });
         });
-        test(`Each object in the  returned array should have keys of 'slug' and 'description'`, () => {
-            return request(app)
-                .get("/api/topics")
-                .expect(200)
-                .then(({ body }) => {
-                    body.forEach((topic) => {
-                        expect(topic).toHaveProperty("slug");
-                        expect(topic).toHaveProperty("description");
-                    });
-                });
+    });
+    test("Each object in the returned array should only have 2 keys", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((topic) => {
+            expect(Object.keys(topic).length).toBe(2);
+          });
         });
-        test("Each object in the returned array should only have 2 keys", () => {
-            return request(app)
-                .get("/api/topics")
-                .expect(200)
-                .then(({ body }) => {
-                    body.forEach((topic) => {
-                        expect(Object.keys(topic).length).toBe(2);
-                    });
-                });
+    });
+  });
+  describe("GET /api/articles/:article_id", () => {
+    test("Should return 200 status code", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+    test("Should return an object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+          expect(typeof body).toBe("object");
         });
-    })
-})
+    });
+    test("Returned object should have desired keys only", () => {
+      const articleTemplate = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+          if (Object.keys(body).length > 1) {
```

Comment:
This if statement is not necessary here, but please make sure you're checking the actual article id, to make sure it's the right article.